python/desc/slrealizer/SLRealizer.py

```diff
@@ -1,27 +1,45 @@
-import plotting
+import desc.slrealizer
 import pandas as pd
 import matplotlib.pyplot as plt
 import pylab
 import matplotlib
 import math
 
 class SLRealizer(object):
-    
+
     scale_factor = 2
-    
+
     # Maybe make a separate method for it
     # Best OOP practice?
 
-    def __init__(self, lens = None, observation = "../../data/twinkles_observation_history.csv"):
+    def __init__(self, catalog=None, observation="../../data/twinkles_observation_history.csv"):
         """
             Reads in a lens sample catalog and observation data.
             We assume lenses are OM10 lenses and observation file is .csv file
         """
-        self.lens = lens
+        self.catalog = catalog
         self.observation = pd.read_csv(observation,index_col=0).as_matrix()
-    
-    def plot_lens_random_date(self, lensID = 7176527, convolve=False):
-        plotting.plot_lens_random_date(self, lensID, convolve)
+
+    # def plot_lens_random_date(self, lensID = 7176527, convolve=False):
+    #     plotting.plot_lens_random_date(self, lensID, convolve)
+
+    def plot_lens_random_date(self, lensID=None, convolve=False):
```

Comment:
It does! Both `plotting.py` and `SLRealizer.py` have functions that is named `plot_lens_random_date`. `SLRealizer.py`'s method `plot_lens_random_date` calls the method `plot_lens_random_date` in `plotting.py`.